### PR TITLE
restruct enable-qe-catalogsource step

### PR DESCRIPTION
**Pull request by jianzhangbjz**
Split step `mirror-images-enable-qe-catalogsource` to:

- enable-qe-catalogsource (for connected env)
- enable-qe-catalogsource-disconnected (for disconnected env)

### CODE REVIEW

**Comment by jianzhangbjz** [comment]
@jianlinliu @yunjiang29 Could you help have a review when you get a chance? Thanks!

File: ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/private/ovn/ipsec/provision/cucushift-installer-rehearse-aws-ipi-private-ovn-ipsec-provision-chain.yaml [hunk 1]
```diff
@@ -13,7 +13,7 @@ chain:
     - ref: ovn-conf
     - ref: ovn-conf-ipsec-manifest
     - ref: ipi-install-install
-    - ref: mirror-images-enable-qe-catalogsource
+    - ref: enable-qe-catalogsource-disconnected
```
Comment (jianlinliu) [inline ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/private/ovn/ipsec/provision/cucushift-installer-rehearse-aws-ipi-private-ovn-ipsec-provision-chain.yaml:16]: This one should use `enable-qe-catalogsource`

Comment (jianzhangbjz) [reply ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/private/ovn/ipsec/provision/cucushift-installer-rehearse-aws-ipi-private-ovn-ipsec-provision-chain.yaml:16]: Thanks! Updated it.

**Pull request merged**